src/ring/middleware/anti_forgery/strategy/session.clj

```diff
@@ -0,0 +1,36 @@
+(ns ring.middleware.anti-forgery.strategy.session
```

Comment:
Move to `ring.middleware.anti-forgery.session` for consistency with `ring.middleware.session`.